README.rst

```diff
@@ -9,7 +9,7 @@ Installation instructions
 
 Create a conda environment for labadmin::
 
-   conda create -n labadmin psycopg2 python=2 click requests
+   conda create -n labadmin psycopg2 python=2 click requests bioconda java-jdk
```

Comment:
nope. I couldn't follow instructions, because my java on Linux Mint is < 1.8, thus I needed to use a conda env, but I might be a special case since I am not using a Mac. Thus, for the default (Mac) user the instructions might work. I simply cannot test them :-/